rkt/main.go

```diff
@@ -0,0 +1,36 @@
+// Copyright 2014 The rkt Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"os"
+
+	"github.com/coreos/rkt/pkg/multicall"
+)
+
+func main() {
+	// check if rkt is executed with a multicall command
+	multicall.MaybeExec()
+
+	cmdRkt.SetUsageFunc(usageFunc)
+
+	// Make help just show the usage
+	cmdRkt.SetHelpTemplate(`{{.UsageString}}`)
+
+	if err := cmdRkt.Execute(); err != nil && cmdExitCode == 0 {
```

Comment:
Interesting, thanks for pointing out. Let's maybe leave a `// err printed by cobra on stderr` then?